api/taxonomies/serializers.py

```diff
@@ -0,0 +1,4 @@
+from rest_framework import serializers as ser
+
```

Comment:
suuuuuper minor: convention is two spaces after imports! Here and in the urls and views files